frontend/src/components/RideModal/RideModal.tsx

```diff
@@ -1,16 +1,20 @@
 import React, { useEffect, useState } from 'react';
```

Comment:
![image](https://user-images.githubusercontent.com/52215742/114092868-61e05c80-9888-11eb-92dd-7f98018ee228.png)
The radio buttons would be better column aligned, (like a table or something) but that's not a big issue.

Clicking next does not display a warning/an error.

Reply:
I had to resolve a merge conflict in order to test the code.

It may be worth doing a quick check on the merge conflicts. Hopefully, I did not break anything in resolving the conflict.

Reply:
nits but could also increase the margin between the radio buttons and the two buttons in the bottom. Also if the drivers page is no longer optional do we still need the unscheduled rides table?